src/ports/catalog/queries.ts

```diff
@@ -265,6 +265,28 @@ const getOwnersJoin = (schemaVersion: string) => {
     )
 }
 
+const getMinPriceCase = (filters: CatalogQueryFilters) => {
+  return SQL`CASE
+                WHEN items.available > 0 AND items.search_is_store_minter = true 
+                `.append(
+    filters.minPrice ? SQL`AND items.price >= ${filters.minPrice}` : SQL``
+  ).append(` THEN LEAST(items.price, nfts_with_orders.min_price) 
+                ELSE nfts_with_orders.min_price 
+              END AS min_price
+            `)
+}
+
+const getMaxPriceCase = (filters: CatalogQueryFilters) => {
+  return SQL`CASE
+                WHEN available > 0 AND items.search_is_store_minter = true 
```

Comment:
Is `available` different from the `items.available` used in the min price case?
Would you mind using the same (and then change the tests)? 🙏

Reply:
sure, added!